table.go

```diff
@@ -226,12 +302,6 @@ func Diff(t1, t2 Table, diffRows bool) (*TableDiff, error) {
 			// Row keys match, compare column values.
 			for _, c := range cmpcols {
```

Comment:
If I followed the code correctly, I think `cmpcols` is the union of `cols1` and `cols2` **minus** `key1` (through `keyMap`). I think it would be more intuitive to compare all columns, including the keys. In the majority of cases where `key1 == key2`, it looks to me like this will not change the output at all. In the edge cases where `key1 != key2`, I think I would want to see the details of what is happening. Even if you disagree, shouldn't we then also be excluding `key2` from `cmpcols`?

Reply:
Yes I didn't change this after I decided to *not* ignore the columns that are added or removed. I will update to make this more clear.

Reply:
Yeah I kind of guessed it might be a leftover from before that change. Are you going to compare the key columns or keep them out?

Reply:
Since the key columns are used to match the record either both tables, by definition they are they are equal 😉